src/test/java/org/folio/mr/api/MediatedRequestActionsApiTest.java

```diff
@@ -433,7 +453,18 @@ void sendItemInTransitFailsWhenMediatedRequestIsNotFoundByStatus() {
   }
 
   @SneakyThrows
-  private ResultActions sendItemInTransit(String itemBarcode) {
+  private ResultActions sendItemInTransit(String itemBarcode, MediatedRequestEntity request) {
+    wireMockServer.stubFor(WireMock.get(urlMatching("/search/instances" + ".*"))
```

Comment:
Does mocking search endpoint really have anything to do with sending item in transit? I'd keep these those two actions separate. Same for `MediatedRequestActionsApiTest#confirmItemArrival`.